src/EA.Weee.Web/Areas/Admin/Controllers/ProducerSubmissionController.cs

```diff
@@ -118,9 +118,14 @@ public async Task<ActionResult> TotalEEEDetails(string registrationNumber, int?
         }
 
         [HttpGet]
+        [AdminSmallProducerSubmissionContext]
```

Comment:
Do we need a unit test for this attribute

Reply:
Added!